css/main.css

```diff
@@ -70,12 +70,24 @@ button{
 
 #toTop{
 	position: fixed;
-	bottom: 95px;
-	right: 40px;
+	bottom: 60px;
+	right: 30px;
 	cursor: pointer;
-	display: none;
 }
-#toTop .fa {margin-right: 5px;}
+
```

Comment:
As explained, these are code changes from my PR which are already merged. 
This is obvious here, but if several people work on the same lines of code, reviewing becomes impossible...